### ISSUE TITLE
False Positive in Layout/EmptyLinesAroundArguments with inline private

### ISSUE THREAD

**Issue by jlfaber** [bbatsov/rubocop#5333]
With the following code fragment using an inline access specifier:
```ruby
class Foo
  private def bar(baz)
    baz

    baz
  end
end
```
## Expected behavior

The Layout/EmptyLinesAroundArguments cop should report no offense.

## Actual behavior

`Layout/EmptyLinesAroundArguments: Empty line detected around arguments.`

## Steps to reproduce the problem

Run rubocop on the fragment above.

The false positive will also be raised when using other inline access specifiers like `protected`, `public`, `private_class_method`, etc. Presence or absence of actual arguments on the method declaration does not matter -- the cop appears to be complaining about the blank line in the method body.  Removing that blank line stops the complaint.

## RuboCop version
```
$ rubocop -V
0.52.1 (using Parser 2.4.0.2, running on ruby 2.4.3 x86_64-darwin16)
```

**Comment by garettarrowood** [comment]
Thanks for reporting this bug!  I'll take a crack at fixing this in the next few days.

**Comment by garettarrowood** [comment]
After looking at this, there actually is not a bug here. `private_class_method` and other inline access modifiers are all methods of [Module](http://ruby-doc.org/core-2.0.0/Module.html#method-i-protected_instance_methods). So the offense getting thrown is because the method `:bar` is an argument of `private()`, and the arg contains an empty line.

We have a test that assures this kind of behavior - https://github.com/bbatsov/rubocop/blob/master/spec/rubocop/cop/layout/empty_lines_around_arguments_spec.rb#L41 .

It looks like this:

```ruby
    it 'registers offense for empty line in-between one arg' do
      inspect_source(<<-RUBY.strip_indent)
        do_stuff([
          "some text",

          "some more text"
        ])
      RUBY
      expect(cop.messages)
        .to eq(['Empty line detected around arguments.'])
    end
```

This is understandable confusing. The example above is both an extra line around arguments, and an extra line inside an array literal. Similarly, the issue example is an extra line around arguments, and an extra line in a method definition.

Should the implementation & possibly cop name be changed to `EmptyLinesBetweenArguments`? -- I lean towards yes.
